src/App.js

```diff
@@ -14,13 +14,15 @@ import Profile from './Profile/Profile';
 import Categories from './Categories/Categories';
 import About from './About/About';
 import HotTakes from './HotTakes/HotTakes';
+import WelcomePage from './LandingPage/Welcome';
 
 const App = () => (
   <NativeRouter>
     <SafeAreaView>
       <View style={styles.main}>
         <Route path="/about" component={About} />
-        <Route exact path="/" component={Categories} />
+        <Route exact path="/" component={WelcomePage} />
+        <Route path="/categories" component={Categories} />
```

Comment:
Thanks for switching this. Categories is now a secondary page and WelcomePage is the root.

Reply:
No problem! Wasn't a tough fix at all, and thought it would save us some headaches in the future.